first_hour_on_terra/workspace_resource_examples.ipynb

```diff
@@ -0,0 +1,1403 @@
+    "            description='Add resource',\n",
+    "            disabled=False,\n",
+    "            button_style='',\n",
+    "            tooltip='Click to add a BigQuery dataset as a controlled resource.',\n",
+    "            icon='plus'\n",
+    "        )\n",
+    "        self.button.on_click(self.create_gcs_bucket)\n",
+    "        self.vb = widgets.VBox(\n",
+    "            [\n",
+    "                self.label,\n",
+    "                self.access_drop_down,\n",
+    "                self.input_auto_delete,\n",
+    "                self.input_bucket_name,\n",
+    "                self.cloning_drop_down,\n",
+    "                self.input_description,\n",
+    "                self.input_lifecycle,\n",
+    "                self.storage_drop_down,\n",
+    "                self.input_name,\n",
+    "            ]\n",
+    "        )\n",
+    "\n",
+    "    def create_gcs_bucket(self,b):\n",
+    "        with self.output:\n",
+    "            description_content = f\"\\\"{self.input_description.value}\\\"\"\n",
+    "\n",
+    "            terraCommand = f\"\"\"terra resource create gcs-bucket \\\\\n",
+    "                --access={self.access_drop_down.value} \\\\\n",
+    "                --auto-delete={self.input_auto_delet.value} \\\\\n",
+    "                --bucket-name={self.input_bucket_name.value} \\\\\n",
+    "                --cloning={self.cloning_drop_down.value} \\\\\n",
+    "                --description={description_content} \\\\\n",
+    "                --name={self.input_name.value} \\\\\n",
+    "                --storage={self.storage_drop_down.value} \\\\\n",
+    "                --workspace={CURRENT_WORKSPACE_ID}\"\"\"     \n",
+    "            print(terraCommand)\n",
+    "            result = subprocess.run([\"terra\",\"resource\",\"create\",\"gcs-bucket\",\n",
+    "                                     f\"--access={self.access_drop_down.value}\",\n",
+    "                                     f\"--auto-delete={self.input_auto_delete.value}\",\n",
+    "                                     f\"--bucket-name={self.input_bucket_name.value}\",\n",
+    "                                     f\"--cloning={self.cloning_drop_down.value}\",\n",
+    "                                     f\"--description={description_content}\",\n",
+    "                                     f\"--name={self.input_name.value}\",\n",
+    "                                     f\"--storage={self.storageDropDown.value}\",\n",
+    "                                     f\"--workspace={CURRENT_WORKSPACE_ID}\"],capture_output=True,text=True)\n",
+    "            print(result.stderr) if not result.stdout else print(result.stdout)\n",
+    "\n",
+    "\n",
+    "create_gcs_bucket_widget = CreateGcsBucketWidget()\n",
+    "display(create_gcs_bucket_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3. Create a GCP notebook\n",
```

Comment:
Even though the command is called `create gcp-notebook`, I really think we should call it a cloud environment. https://terra-docs.api.verily.com/docs/how_to_guides/cloud_environments/#accessing-the-terra-command-line-tool-from-your-cloud-environment for example correctly calls it a cloud environment even though the command line example calls it a gcp-notebook.

In fact, I'm going to file a ticket to rename it in the CLI.